public/index.html

```diff
@@ -2,7 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/logo.png" />
+    <link rel="stylesheet" href="./styles.css">
```

Comment:
Лучше использовать следующий путь:
```suggestion
    <link rel="stylesheet" href="%PUBLIC_URL%/styles.css">
```

При сборке он подставится на корректный